### PR TITLE
Stackdriver Uptime Check sample.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

File: monitoring/api/AlertSample/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+This sample requires [.NET Core 2.0](
+    https://www.microsoft.com/net/core) or later.  That means using
+[Visual Studio 2017](
+    https://www.visualstudio.com/), or the command line.  Visual Studio 2015 users
+can use [this older sample](
+    https://github.com/GoogleCloudPlatform/dotnet-docs-samples/tree/vs2015/monitoring/api).
+
+## Build and Run
+
+1.  **Follow the instructions in the [root README](../../README.md)**.
```
Comment (jsimonweb) [inline monitoring/api/AlertSample/README.md]: Update markdown href path 
    [root README](../../README.md)
to be
    [root README](../../../README.md)

File: monitoring/api/MonitoringSample/README.md [hunk 2]
```diff
@@ -18,18 +18,8 @@ can use [this older sample](
     [Click here](https://console.cloud.google.com/flows/enableapi?apiid=monitoring.googleapis.com&showconfirmation=true)
```
Comment (jsimonweb) [inline monitoring/api/MonitoringSample/README.md]: Update markdown href path to root README.md file:
    [root README](../../README.md)
to be
    [root README](../../../README.md)

File: monitoring/api/AlertSample/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# Google Stackdriver Monitoring Sample
+
+This C# sample demonstrates how to manage Stackdriver Alerts using
+[Google Stackdriver Monitoring API](https://cloud.google.com/monitoring/alerts/using-alerting-api) from C#.
+
+This sample requires [.NET Core 2.0](
+    https://www.microsoft.com/net/core) or later.  That means using
+[Visual Studio 2017](
+    https://www.visualstudio.com/), or the command line.  Visual Studio 2015 users
```
Comment (simonz130) [inline monitoring/api/AlertSample/README.md]: Should we also mention VS Code?

File: monitoring/api/UptimeCheckSample/Program.cs [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+                },
+                Timeout = TimeSpan.FromSeconds(10).ToDuration(),
+                Period = TimeSpan.FromMinutes(5).ToDuration()
+            };
+            // Create a client.
+            var client = UptimeCheckServiceClient.Create();
+            string projectName = new ProjectName(projectId).ToString();
+            // Create the config.
+            var newConfig = client.CreateUptimeCheckConfig(projectName, config);
+            Console.WriteLine(newConfig.Name);
```
Comment (simonz130) [inline monitoring/api/UptimeCheckSample/Program.cs]: nit: maybe you can be a bit more descriptive in the output message

File: monitoring/api/UptimeCheckSample/Program.cs [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+            return 0;
+        }
+        // [END monitoring_uptime_check_create]
+
+        // [START monitoring_uptime_check_delete]
+        public static object DeleteUptimeCheckConfig(string configName)
+        {
+            var client = UptimeCheckServiceClient.Create();
+            client.DeleteUptimeCheckConfig(configName);
```
Comment (simonz130) [inline monitoring/api/UptimeCheckSample/Program.cs]: what happens if the config does not exist?

File: monitoring/api/UptimeCheckSample/Program.cs [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+            {
+                Console.WriteLine("{0,20} {1}", ip.IpAddress, ip.Location);
+            }
+            return 0;
+        }
+        // [END monitoring_uptime_check_list_ips]
+
+        // [START monitoring_uptime_check_update]
+        public static object UpdateUptimeCheck(string configName,
+            string newHttpPath, string newDisplayName)
```
Comment (simonz130) [inline monitoring/api/UptimeCheckSample/Program.cs]: We should document the format of this path as well as what is the role of this parameter here.

File: monitoring/api/UptimeCheckSample/Program.cs [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+        {
+            var client = UptimeCheckServiceClient.Create();
+            var config = client.GetUptimeCheckConfig(configName);
+            var fieldMask = new FieldMask();
+            if (newDisplayName != null)
```
Comment (simonz130) [inline monitoring/api/UptimeCheckSample/Program.cs]: Should we remove the input parameters validation or add it in other snippets to make it more consistent across the sample code?

File: monitoring/api/AlertSample/README.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+11. Run the AlertSample:
+    ```
+    PS C:\...\dotnet-docs-samples\monitoring\api\AlertSample> dotnet run
+    AlertSample 1.0.0
+    Copyright (C) 2018 AlertSample
+
+    ERROR(S):
+    No verb selected.
+
+    list                List alert policies.
```
Comment (simonz130) [inline monitoring/api/AlertSample/README.md]: We shouldn't block this sample code on this, but going forward, code sample working group decided on not having command lines for snippets.

**Pull request merged**